azure-kusto-data/azure/kusto/data/helpers.py

```diff
@@ -48,33 +80,25 @@ def dataframe_from_result_table(table: "Union[KustoResultTable, KustoStreamingRe
 
     columns = [col.column_name for col in table.columns]
     frame = pd.DataFrame(table.raw_rows, columns=columns)
+    default = default_dict()
+
+    if nullable_bools:
```

Comment:
Don't copy the entire dict for that.

Just use an if when getting from the default dict